### ISSUE TITLE
Filename pattern is not working

### ISSUE THREAD

**Issue by madsem** [armin-pfaeffle/atom-minify#42]
I have tried several different settings, none did anything. Minified files are always saved to the directory the original file resides in.

For example:
```
../$1.min.$2
```

Expected result: file will be saved one dir up from the dir where the current file sits
result: File is saved in the same dir where current file resides.

Same result with any other setting I've tried:

```
../css/min/$1.min.$2
```
```
absolute/path/from/project/root/css/min/$1.min.$2
```

**Comment by armin-pfaeffle** [comment]
Did you set this in the settings? Or do you use a first-line-comment? If yes, can you paste it here?

**Comment by madsem** [comment]
I've set it in the settings exactly as above.
Is there another way to achieve this?

**Comment by armin-pfaeffle** [comment]
Yep! You could try to add this comment in the first line of your file:
```
/* filenamePattern: ../$1.compressed.$2 */
```

I will check your bug report!

**Comment by madsem** [comment]
awesome, thank you. Will let you know tomorrow if the comment worked

**Comment by madsem** [comment]
The filepattern comment works like a charm Armin, granted it's not as comfy as having it in the settings and forget about it, but will do the trick for the time being :)

**Comment by armin-pfaeffle** [comment]
Hm... I tried relative and absolute paths. Everythings works fine here, with every minifier.

Which version of atom-minify do you use? Which minifier? Mac? Win? Linux?

**Comment by Sineos** [comment]
Having a similar issue:

OS: Win 10
Minifier: 0.8.0

`// minifier: gcc, buffer: 8388608, minifierOptions: "compilation_level=SIMPLE_OPTIMIZATIONS externs=D:\21_Git\MD\_tools\lin\FileSaver.min.ext.js externs=D:\21_Git\MD\_tools\lin\jquery.ext.js"`

works, where as

`// minifier: gcc, buffer: 8388608, minifierOptions: "compilation_level=SIMPLE_OPTIMIZATIONS externs=./FileSaver.min.ext.js externs=./jquery.ext.js"`

will throw an error.
